### PR TITLE
Rajivnan/msp dotnet ga

### PR DESCRIPTION
**Pull request by rajivnandivada**
Updating the sdk for the new version 2019-06-01.
Corresponding REST API spec PR https://github.com/Azure/azure-rest-api-specs/pull/6406

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -0,0 +1,11 @@
+{
```
Comment (erich-wang) [inline package-lock.json]: is this file added accidentally?

File: sdk/managedservices/Microsoft.Azure.Management.ManagedServices/AzSdk.RP.props [hunk 2]
```diff
@@ -1,7 +1,7 @@
 <Project ToolsVersion="15.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <!--This file and it's contents are updated at build time moving or editing might result in build failure. Take due deligence while editing this file-->
   <PropertyGroup>
-    <AzureApiTag>ManagedServices_2018-06-01-preview;</AzureApiTag>
+    <AzureApiTag>ManagedServices_2019-06-01;</AzureApiTag>
```
Comment (erich-wang) [inline sdk/managedservices/Microsoft.Azure.Management.ManagedServices/AzSdk.RP.props]: Please update the value of **Version** in Microsoft.Azure.Management.ManagedServices.csproj

**Review by erich-wang — CHANGES_REQUESTED** [review]

File: package-lock.json (hunk 1 above)
Comment (rajivnandivada) [reply package-lock.json]: yes, sorry.  I will remove it.

**Comment by rajivnandivada** [comment]
@erich-wang I have addressed your feedback. Please review.

**Review by erich-wang — APPROVED** [review]

**Pull request merged**